common/errors/utils_test.go

```diff
@@ -0,0 +1,106 @@
+package errors
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestErrorStack(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		err     error
+		contain string
+	}{
+		{
+			err:     New("foo"),
+			contain: "New(\"foo\")",
+		}, {
+			err:     fmt.Errorf("bar"),
+			contain: "bar",
+		},
+	}
```

Comment:
please add one more case, to make sure `Unwrap` also tested. 
```go
fmt.Errorf("wrap another error: %w", errors.New("inner error")) 
```
BTW why I use `%w` instead of `%s` or  `%v`, https://blog.golang.org/go1.13-errors#TOC_3.3.

Reply:
ok got it. updated